documentation-site/next.config.js

```diff
@@ -71,6 +72,12 @@ module.exports = withTM(
       webpack: (config, {buildId, dev, isServer, defaultLoaders}) => {
         config.resolve.alias.baseui = resolve(__dirname, '../dist');
         config.resolve.alias.examples = resolve(__dirname, 'examples');
+        // Force a non esm bundle of d3.
+        // Can remove once we upgrade to next 11 or 12 which has support for es modules.
+        config.resolve.alias.d3 = resolve(
```

Comment:
Next.js doesn't support 3rd part esm modules, have we evaluated this with the latest nextjs?